tests/helpers/perf_run_helper.rb

```diff
@@ -23,6 +23,7 @@ module PerfRunHelper
                                       else
                                         ["puppetserver"]
                                       end
+  PUPPET_METRICS_COLLECTOR_DIR_NAME = "puppet-metrics-collector"
```

Comment:
Since PerfResultsHelper is included on line 11 and this constant is declared there, this line should not be needed.

Reply:
Good call; I will try removing it and re-test.